actions/types.go

```diff
@@ -0,0 +1,144 @@
+package actions
+
+import (
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/rds"
+)
+
+type DatabaseCreateRequest struct {
+	Cluster  *CreateDBClusterInput
+	Instance *CreateDBInstanceInput
+}
+
+func (dcr DatabaseCreateRequest) String() string {
+	s, _ := json.MarshalIndent(dcr, "", "\t")
+	return string(s)
+}
```

Comment:
You define this here but in the orchestrator you use `%+v` - you probably don't need this.

Reply:
It works. Without it I get `&{Cluster:0xc0003560c0 Instance:<nil>}` when I print the struct

Reply:
oh that's interesting...  I'll have to try that.  i always use `awsutil.PrettyPrint()` when I want those details, maybe that's all it does. 🤯